ddl/db_test.go

```diff
@@ -516,8 +514,9 @@ LOOP:
 			// delete some rows, and add some data
 			for i := count; i < count+step; i++ {
 				n := rand.Intn(count)
-				// Don't delete this row, otherwise error message would change.
-				if n == defaultBatchSize*2-10 {
+				// (2048, 2038, 2038) and (2038, 2038, 2038)
```

Comment:
I don't get the comment here. Could you add an explanation with more details?

Reply:
These are the two duplicate rows. So we shouldn't delete either of them.

Reply:
count += step, so it may > 2048.
This test wants to randomly delete some rows. So it doesn't remove data in the range of [0, 2037] directly.
It's not easy to understand, that why I missed this check at the first fix.